src/ExpressionLanguage/ExpressionFunction/GraphQL/Relay/IdFetcherCallback.php

```diff
@@ -18,6 +19,10 @@ function ($idFetcher) {
                 $code .= 'return '.$idFetcher.'; }';
 
                 return $code;
+            },
+            // This expression function is not designed to be used by it's evaluator
+            function (): void {
```

Comment:
you can use `new EvaluatorIsNotAllowedException($name)` since Exception is callable

Reply:
Done!